src/tests/test_magic.py

```diff
@@ -2635,3 +2637,17 @@ def test_table_does_not_exist_with_snippet_error(
 def test_negative_operations_query(ip, query, expected):
     result = ip.run_cell(query).result
     assert list(result.dict().values())[-1] == expected
+
+
+@pytest.mark.parametrize(
```

Comment:
Please add testcase for message `Named parameters are currently disabled. If you wish to use named parameters, enable them with...`

Reply:
[Added testcase for disable message](https://github.com/ploomber/jupysql/pull/980/files#diff-d327143929530bcece2cae65d59465708cdf87f499fd83b4bd939c332150f713R2656-R2672)